themes/src/main/resources-community/theme/base/account/messages/messages_es.properties

```diff
@@ -144,4 +201,158 @@ invalidPasswordMinUpperCaseCharsMessage=Contrase\u00F1a incorrecta: debe contene
+invalidPasswordGenericMessage=Contrase\u00F1a no v\u00E1lida: La nueva contrase\u00F1a no cumple con las politicas de contrase\u00F1a.
+
+# Authorization
+myResources=Mis recursos
+myResourcesSub=Mis recursos
+doDeny=Denegar
+doRevoke=Revocar
+doApprove=Aprovar
+doRemoveSharing=Eliminar compartir
+doRemoveRequest=Eliminar solicitud
+peopleAccessResource=Personas con acceso a este recurso.
+resourceManagedPolicies=Permisos que otorgan acceso a este recurso
+resourceNoPermissionsGrantingAccess=No hay permisos que otorguen acceso a este recurso
+anyAction=Cualquier acci\u00F3n
+description=Descripci\u00F3n
+name=Nombre
+scopes=Alcances
+resource=Recurso
+user=Usuario
+peopleSharingThisResource=Personas que comparten este recurso
+shareWithOthers=Compartir con otros
+needMyApproval=Necesita mi aprobaci\u00F3n
+requestsWaitingApproval=Sus solicitudes en espera de aprobaci\u00F3n
+icon=Icono
+requestor=Solicitante
+owner=Propietario
+resourcesSharedWithMe=Recursos compartidos conmigo
+permissionRequestion=Solicitud de permiso
+permission=Permiso
+shares=compartido(s)
+notBeingShared=Este recurso no se comparte.
+notHaveAnyResource=Usted no tiene recursos
+noResourcesSharedWithYou=No hay recursos compartidos con usted
+havePermissionRequestsWaitingForApproval=Usted tiene {0} solicitudes de permiso en espera de aprobaci\u00F3n.
+clickHereForDetails=Haga clic aqu\u00ED para m\u00E1s detalles.
+resourceIsNotBeingShared=El recurso no se comparte
+
+locale_ca=Catal\u00e0
+locale_de=Deutsch
+locale_en=English
+locale_es=Espa\u00f1ol
+locale_fr=Fran\u00e7ais
+locale_it=Italian
+locale_ja=\u65e5\u672c\u8a9e
+locale_nl=Nederlands
+locale_no=Norsk
+locale_lt=Lietuvi\u0173
+locale_pt-BR=Portugu\u00eas (Brasil)
+locale_ru=\u0420\u0443\u0441\u0441\u043a\u0438\u0439
+locale_sk=Sloven\u010dina
+locale_sv=Svenska
+locale_tr=Turkish
+locale_zh-CN=\u4e2d\u6587\u7b80\u4f53
+
+# Applications
+applicaitonName=Nombre
+applicationType=Tipo de aplicaci\u00F3n
+applicationInUse=In-use app only
+clearAllFilter=Lipiar todos los filtros
+activeFilters=Filtros activos
+filterByName=Filtar por nombre
+allApps=Todas las aplicaciones
+internalApps=Aplicaciones internas
+thirdpartyApps=Aplicaciones de terceros
+appResults=Resultados
+clientNotFoundMessage=Cliente no encontrado.
+
+# Linked account
+authorizedProvider=Proveedor autorizado
+authorizedProviderMessage=Proveedores autorizados vinculados a su cuentat
+identityProvider=Proveedores de identidad
+identityProviderMessage=Para vincular su cuenta con proveedores de identidad que haya configurado
+socialLogin=Iicio de sesi\u00F3n con redes sociales
+userDefined=Definido por el usuario
+removeAccess=Eliminar acceso
+removeAccessMessage=Deber\u00E1 volver a otorgar acceso si desea utilizar esta cuenta de aplicaci\u00F3n.
+
+#Authenticator
+authenticatorStatusMessage=La autenticaci\u00F3n de dos factores esta activa actualmente
```

Comment:
Here I see in the English version that does not specifies the state, just "...dos factores esta actualmente"

Reply:
I would suggest "La autenticación en dos pasos está activada".